config/locales/models/budget.en.yml

```diff
@@ -47,6 +47,7 @@ en:
     budgets:
       button:
         create: Add budget
+      no_budgets: There are no budgets created during this reporting cycle.
```

Comment:
Should this say "created or updated", in line with what the "what it shows" copy says? (If a budget had been created during *another* reporting cycle, and updated during the current one, would it be listed?)

Reply:
I've just had a look, and the budget is always tied to the report, so the introductory copy is wrong! Will fix 👍